config/boards/helios64.conf

```diff
@@ -4,7 +4,10 @@ BOARD_VENDOR="kobol"
 BOARDFAMILY="rockchip64" # Used to be rk3399
 BOARD_MAINTAINER="prahal"
 BOOTCONFIG="helios64-rk3399_defconfig"
-BOOT_SCENARIO="tpl-blob-atf-mainline"
+BOOTBRANCH="tag:v2026.01-rc5"
+BOOTPATCHDIR="v2026.01"
```

Comment:
Given you've quite a bit of custom patching here, maybe use a dedicated directory for this board? That way you can have your own 000.patching.config.yaml in there without impacting other v2026.01 u-boots.

For an example see `patch/u-boot/v2024.04/board_t95z`; the board file has `BOOTPATCHDIR="v2024.04/board_t95z"`

Reply:
> use a dedicated directory for this board? That way you can have your own 000.patching.config.yaml in there without impacting other v2026.01 u-boots.

done